.github/scripts/count-points.mjs

```diff
@@ -4,43 +4,32 @@ import pLimit from 'p-limit';
 
 const limit = pLimit(50);
 
-/** Repository-specific configuration for points calculation */
-const REPOSITORY_CONFIG = new Map([
-  [
-    'actual',
-    {
-      POINTS_PER_ISSUE_TRIAGE_ACTION: 1,
-      POINTS_PER_ISSUE_CLOSING_ACTION: 1,
-      POINTS_PER_RELEASE_PR: 0,
-      PR_REVIEW_POINT_TIERS: [
-        { minChanges: 500, points: 8 },
-        { minChanges: 100, points: 6 },
-        { minChanges: 10, points: 2 },
-        { minChanges: 0, points: 1 },
-      ],
-      EXCLUDED_FILES: [
-        'yarn.lock',
-        '.yarn/**/*',
-        'packages/component-library/src/icons/**/*',
-        'release-notes/**/*',
```

Comment:
so it turns out this wasn't working, the folder we're looking for is `upcoming-release-notes`, not `release-notes` so some PRs had their points inflated (only by one at most in the cases I've found)

This fix changes that, so some points totals from last month are expected to change slightly.

Reply:
Cases I've found:

#6005
#5989
#5869
#5848
#5878
#5885
#5853
#5936
#5807